.gitignore

```diff
@@ -1,4 +1,35 @@
-coverage
-dist
-node_modules
-tsconfig.tsbuildinfo
+# compiled output
```

Comment:
Better to put the noise about logs and IDE in a global .gitignore :man_shrugging: